cmd/cnab-to-oci/pull.go

```diff
@@ -30,8 +31,9 @@ func pullCmd() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.output, "output", "o", "pulled.json", "output file")
+	cmd.Flags().StringVarP(&opts.output, "output", "o", "pulled.json", "Output file")
 	cmd.Flags().StringSliceVar(&opts.insecureRegistries, "insecure-registries", nil, "Use plain HTTP for those registries")
+	cmd.Flags().BoolVarP(&opts.raw, "raw", "", false, "If true, pull and write the raw manifest from the registry")
```

Comment:
What about renaming this option to `--manifest` ? I think it's more explicit than `--raw` 🤔 ?

Reply:
Sure, it totally makes more sense.
Updating.